modules/mri_violations/help/mri_violations.md

```diff
@@ -0,0 +1,15 @@
+# MRI Protocol Violations
```

Comment:
The Menu Item is `MRI Violated Scans` should this help file show the same label/header?